ocaml/quicktest/quicktest_vm_snapshot.ml

```diff
@@ -0,0 +1,92 @@
+(** Set up snapshot test: create a small VM with a selection of VBDs *)
+let with_setup rpc session_id sr vm_template f =
+  print_endline "Setting up test VM" ;
+  let uuid = Client.Client.VM.get_uuid rpc session_id vm_template in
+  print_endline (Printf.sprintf "Template has uuid: %s%!" uuid) ;
+  Qt.VM.with_new rpc session_id ~template:vm_template (fun vm ->
+      print_endline (Printf.sprintf "Installed new VM") ;
+      print_endline
+        (Printf.sprintf "Using SR: %s"
+           (Client.Client.SR.get_name_label ~rpc ~session_id ~self:sr)
+        ) ;
+      let vdi =
+        Client.Client.VDI.create rpc session_id "small" "description" sr
```

Comment:
Instead of description `__LOC__` provides a more useful string that would help debug this. Same below.